spec/models/shelter_spec.rb

```diff
@@ -67,5 +68,18 @@
         expect(@shelter_1.pet_count).to eq(3)
       end
     end
+
+    describe ".reverse_abc" do
+      it "returns the shelters in reverse order" do
+        expect(Shelter.reverse_abc).to eq([@shelter_2, @shelter_3, @shelter_1])  
+      end
+    end
+    
+    describe ".pending_applications" do
+      it "returns the pending applications" do 
+       
+        expect(Shelter.pending_applications).to eq([])
```

Comment:
This is not the most robust test because the test data provided isn't extensive enough. This doesn't verify that the query you have is returning the correct thing since it's returning an empty array. It should probably return the @application you have in your test data, no? Or you can create another test application and see what the output is

Reply:
You'll also need to create some ApplicationPets in order to this application to be associated to a shelter